tests/fixtures/config/config.yml

```diff
@@ -11,6 +11,10 @@ runtime:
         # Write to log file every N seconds
         log_interval: 5
 
+    versioning:
+        runtime_image: "1.2.3"
+        foo: bar
+
```

Comment:
I don’t fully understand the flow of how a user sets details in `config.yml` to be populated in the endpoint, what type of info would this use and how does this work for flowing into RPC..?